### PR TITLE
Running boot2docker ip inside boot2docker shell in windows returns not fo...

### PR DESCRIPTION
**Pull request by jaycode**
...und error, let's change it.

### CODE REVIEW

File: docs/sources/userguide/usingdocker.md [hunk 1]
```diff
@@ -181,8 +181,8 @@ Our Python application is live!
 > **Note:**
 > If you have used the boot2docker virtual machine on OS X, Windows or Linux,
 > you'll need to get the IP of the virtual host instead of using localhost.
-> You can do this by running the following in
-> the boot2docker shell.
+> You can do this by running the following outside of
+> the boot2docker shell (i.e. your command line or terminal application).
```
Comment (moxiegirl) [inline docs/sources/userguide/usingdocker.md]: Either Boot2Docker for the product or `boot2docker` for the service reference

Comment (fredlf) [inline docs/sources/userguide/usingdocker.md]: Nice catch. Thanks. 
Sorry to nit pick, but you need a comma after "i.e.". We might also make things a little clearer by rearranging the noun strings: (i.e., from your terminal or command-line application). Just a suggestion... thanks for the contribution.

Comment (jaycode) [inline docs/sources/userguide/usingdocker.md]: Great, thanks.

**Comment by moxiegirl** [comment]
ping @jaycode Would you like us to pick this up for you?

**Comment by jaycode** [comment]
Hi, yes please, I got my hands tied on another project atm. Great job with
docker btw love it so much!

**Comment by icecrime** [comment]
Thanks @jaycode! Carried in #12057 (ping @moxiegirl @fredlf)

**Pull request closed without merging**